x-pack/plugin/sql/jdbc/src/main/java/org/elasticsearch/xpack/sql/jdbc/InfoResponse.java

```diff
@@ -12,10 +12,17 @@
     final String cluster;
     final int majorVersion;
     final int minorVersion;
+    final int revisionVersion;
 
-    InfoResponse(String clusterName, byte versionMajor, byte versionMinor) {
+    InfoResponse(String clusterName, byte versionMajor, byte versionMinor, byte revisionVersion) {
         this.cluster = clusterName;
         this.majorVersion = versionMajor;
         this.minorVersion = versionMinor;
+        this.revisionVersion = revisionVersion;
+    }
+
+    @Override
+    public String toString() {
+        return "v" + majorVersion + "." + minorVersion + "." + revisionVersion;
```

Comment:
The toString should contain the cluster name - if the version is needed, make a separate method (`version` or `versionString`).
`return clusterName[major.minor.revision]`